CMakeLists.txt

```diff
@@ -88,12 +98,8 @@ if (MSVC)
 	endif()
 
 	if (USE_LTO)
-		set_cxx_flag("/GL" RELEASE)
-		set_cxx_flag("/GL" RELWITHDEBINFO)
```

Comment:
The point of these is to have different flags per configuration in a multi-configuration build, so that you can test a Release build with LTO and a Debug build without it from the same IDE instance.

It's OK to remove this for the GCC-style options since it gets in the way of flag tests, but there is no need to remove it for MSVC.

Reply:
OK, this looked overly verbose to me but I didn't know MSVC required such explicit boilerplate.
Now reverted.